package.json

```diff
@@ -1173,7 +1173,7 @@
     "pretty-ms": "6.0.0",
     "prop-types": "^15.8.1",
     "proxy-from-env": "1.0.0",
-    "puppeteer": "22.13.1",
+    "puppeteer": "23.3.0",
```

Comment:
Looks like 23.3.1 is already available: https://github.com/puppeteer/puppeteer/releases